controllers/clustergroupupgrade_controller.go

```diff
@@ -222,7 +222,7 @@ func (r *ClusterGroupUpgradeReconciler) Reconcile(ctx context.Context, req ctrl.
 				meta.SetStatusCondition(&clusterGroupUpgrade.Status.Conditions, metav1.Condition{
 					Type:    "Ready",
 					Status:  metav1.ConditionFalse,
-					Reason:  "UpgradeCannotStart",
+					Reason:  "UpgradeBlockedOnCR",
```

Comment:
UpgradeBlockedOnCR is not accurate for this case - cannot start due to missing policies.

Probably something like "UpgradeNotReadyToStart"?

Reply:
Huh, yes, I did a find/replace.
So this means this particular UpgradeCannotStart has had a dual meaning all this time.
I need to think what's the best transition here.